src/contracts/MetricToken.sol

```diff
@@ -1,11 +1,11 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
+pragma solidity 0.8.13;
 
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 
 /// @custom:security-contact contracts@metricsdao.com
 contract MetricToken is ERC20 {
-    constructor(address _vestingContractAddress) ERC20("METRIC", "METRIC") {
```

Comment:
I did this because as James pointed out we will deploy Metric before we deploy the vesting contract